### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by AnkitaSikdar005**
## What does this PR do?
Add resource(s) 

## For resources
Added C language lectures in Hindi.

### Why is this valuable (or not)?
Yes, it is valuable.
### How do we know it's really free?
It is available on YouTube.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AnkitaSikdar005** [comment]
@eshellman Please merge my pull request.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -135,11 +135,13 @@
 ### <a id="c"></a>C
 
 * [All C Concepts \| Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
+* [C Language for GATE ](https://www.youtube.com/playlist?list=PLC36xJgs4dxG-IqARhc23jYTDMYt7yvZP) - Gate cse lectures by Amit Khurana
 * [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://www.youtube.com/watch?v=irqbmMNs2Bo) - Apna College
 * [C Language Tutorial For Beginners In Hindi (With Notes)](https://www.youtube.com/watch?v=ZSPZob_1TOk) - CodeWithHarry
 * [C Language Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtksgReOH3s7R1_cb1QA8vrb) - codeitup
 * [C Language Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - CodeWithHarry
 * [C Programming Tutorials](https://www.youtube.com/playlist?list=PLiOa6ike4WAHH3HyPUu6pUG-r0LApvW-l) - Vikas Pandey, easytuts4you
+
```
Comment (eshellman) [inline courses/free-courses-hi.md:144]: extra space

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -135,11 +135,13 @@
 ### <a id="c"></a>C
 
 * [All C Concepts \| Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
+* [C Language for GATE ](https://www.youtube.com/playlist?list=PLC36xJgs4dxG-IqARhc23jYTDMYt7yvZP) - Gate cse lectures by Amit Khurana
```
Comment (eshellman) [inline courses/free-courses-hi.md:138]:
```suggestion
* [C Language for GATE](https://www.youtube.com/playlist?list=PLC36xJgs4dxG-IqARhc23jYTDMYt7yvZP) - GATE cse lectures by Amit Khurana
```

**Comment by AnkitaSikdar005** [comment]
@eshellman Made the necessary changes .Please merge this pull request.

File: courses/free-courses-hi.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-hi.md:144]: remove the extra space

**Pull request closed without merging**